engine/previews/citizens_advice_components/header_preview/full_example.html.haml

```diff
@@ -0,0 +1,9 @@
+= render CitizensAdviceComponents::Header.new do |c|
+  :ruby
+    c.logo(title: "Citizens Advice homepage", url: "/")
+    c.skip_links(skip_links)
+    c.header_links(header_links)
+    c.search_form(search_action_url: "/search")
+  - c.account_link do
+    = form_tag("/sign-out/out", authenticity_token: false) do
```

Comment:
This `authenticity_token: false` is important for previews, so we always generate stable HTML. But it's a good thing as it proves that we can use the regular form helpers like `form_tag` and `button_to` and rails will inject `authenticity_token` if needed.